### PR TITLE
use release version v3.0 in recipe

### PR DESCRIPTION
**Pull request by dnadeau4**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,8 +18,8 @@ analysis and simulation.
 Current build status
 ====================
 
-Linux: [![Circle CI](https://circleci.com/gh/conda-forge/cdms2-feedstock.svg?style=shield)](https://circleci.com/gh/conda-forge/cdms2-feedstock)
-OSX: [![TravisCI](https://travis-ci.org/conda-forge/cdms2-feedstock.svg?branch=master)](https://travis-ci.org/conda-forge/cdms2-feedstock)
+Linux: [![Circle CI](https://circleci.com/gh/conda-forge/cdms2-feedstock-dnadeau4-feedstock.svg?style=shield)](https://circleci.com/gh/conda-forge/cdms2-feedstock-dnadeau4-feedstock)
```
Comment (CJ-Wright) [inline README.md]: That doesn't look right

Comment (dnadeau4) [reply README.md]: I agree.  I used `conda smithy rerender -c auto` in my forked repository.

**Comment by CJ-Wright** [comment]
@conda-forge-admin, please rerender

File: README.md (hunk 1 above)
Comment (CJ-Wright) [reply README.md]: Lets see what the webservices do

File: README.md [hunk 2]
```diff
@@ -18,8 +18,8 @@ analysis and simulation.
 Current build status
 ====================
 
-Linux: [![Circle CI](https://circleci.com/gh/conda-forge/cdms2-feedstock-dnadeau4-feedstock.svg?style=shield)](https://circleci.com/gh/conda-forge/cdms2-feedstock-dnadeau4-feedstock)
-OSX: [![TravisCI](https://travis-ci.org/conda-forge/cdms2-feedstock-dnadeau4-feedstock.svg?branch=master)](https://travis-ci.org/conda-forge/cdms2-feedstock-dnadeau4-feedstock)
+Linux: [![Circle CI](https://circleci.com/gh/conda-forge/cdms2-feedstock.svg?style=shield)](https://circleci.com/gh/conda-forge/cdms2-feedstock)
+OSX: [![TravisCI](https://travis-ci.org/conda-forge/cdms2-feedstock.svg?branch=master)](https://travis-ci.org/conda-forge/cdms2-feedstock)
```
Comment (dnadeau4) [inline README.md]: That command worked...

**Pull request merged**